reference/common/models/alertsconfig.v1.yaml

```diff
@@ -0,0 +1,16 @@
+type: object
+properties:
```

Comment:
I think this works well given that there is likely a finite set of possible alerts.

If (that's a big if) we thought that this could expand significantly in the future (i.e. addition types of alerts), we _could_ consider defining this as an array of different alert configurations (in which case each alert would have to have a `type` field to distinguish them). That said, we could just as easily add more named properties to this type.

To be clear, I do **not** think turning this into an array is worth the extra flexibility.

Reply:
Nod. That was discussed at some point, and we landed on named properties exactly for those reasons. I could probably search up the convo, but since you're not asking for an switch to an array, I won't bother just now.